deployment/docker/alerts.yml

```diff
@@ -91,6 +91,19 @@ groups:
             Having less than 20% of free disk space could cripple merges processes and overall performance.
             Consider to limit the ingestion rate, decrease retention or scale the disk space if possible."
 
+      - alert: InsufficientDiskSpaceForMerges
+        expr: |
+          vm_merge_need_free_disk_space > 0
```

Comment:
I don't think this alert is needed. It can trigger when `-storageDataPath` directory has 49% of free space if there are two parts with sizes 25.5% of disk space. These parts may be eligible for merge, but they cannot be merged, since the destination part size will be equal to 51% of disk space. So the merge won't happen while `vm_merge_need_free_disk_space` metric will be greater than 0.

Reply:
removed